src/menu/MenuSongSelection.as

```diff
@@ -112,6 +113,9 @@ package menu
 
         override public function init():Boolean
         {
+            // Listen for DB Connection
+            _gvars.addEventListener(GlobalVariables.DB_CONNECT_COMPLETE, e_sqlConnectComplete);
+
```

Comment:
```suggestion
            // Listen for DB Connection
            _gvars.addEventListener(GlobalVariables.DB_CONNECT_COMPLETE, e_sqlConnectComplete);
            _gvars.addEventListener(GlobalVariables.DB_CONNECT_ERROR, e_sqlConnectError);
```

Reply:
We actually don't need to listen for a DB connection error here, as we only care if it connects successfully.